compiler/rustc_target/src/spec/solid_base.rs

```diff
@@ -0,0 +1,21 @@
+use super::LinkerFlavor;
+use crate::spec::{RelroLevel, TargetOptions};
+
+pub fn opts(kernel: &str) -> TargetOptions {
+    TargetOptions {
+        os: format!("solid-{}", kernel),
+        vendor: "kmc".to_string(),
+        exe_suffix: ".out".to_string(),
+        linker_is_gnu: true,
+        linker_flavor: LinkerFlavor::Gcc,
+        has_rpath: false,
+        crt_static_default: true,
+        crt_static_respected: true,
```

Comment:
Is this target supposed to be statically or dynamically linked. If dynamically, crt-static is likely incorrect as that would cause more than one copy of the C runtime to be loaded into the program.

Reply:
Sort of both, actually. However, it turned out that there are several issues regarding the interaction with the platform linker. Thus, for the time being, we decided to disable dynamic linking by commit 655f36606dc926c99aa42bd5c12085844c2e2981.